src/lib/services/events.js

```diff
@@ -76,8 +77,20 @@ const getMostRecentReturnsInvitationByLicence = async licenceRef => {
   return repo.events.getMostRecentReturnsInvitationByLicence(licenceRef);
 };
 
+const getKPIReturnsMonthly = async () => {
+  const result = await repo.events.getKPIReturnsMonthlyData();
+  return result.rowCount > 0 ? [...camelCase(result.rows)] : null;
+};
+
+const getKPILicenceNames = async () => {
+  const result = await repo.events.getKPILicenceNamesData();
+  return result.rowCount > 0 ? [...camelCase(result.rows)] : null;
+};
+
```

Comment:
Perhaps we need something like the following to help appease code climate

```
const nullIfEmpty = (arr = []) => arr.length ? arr : null;

const getKPIReturnsMonthly = async () => {
  const result = await repo.events.getKPIReturnsMonthlyData();
  return nullIfEmpty(camelCase(result.rows));
};

const getKPILicenceNames = async () => {
  const result = await repo.events.getKPILicenceNamesData();
  return nullIfEmpty(camelCase(result.rows));
};

```

Reply:
done